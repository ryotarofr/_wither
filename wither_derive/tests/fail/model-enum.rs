use serde::{Serialize, Deserialize};
use wither::Model;

#[derive(Serialize, Deserialize, Model)]
enum BadModel {}

fn main() {}
