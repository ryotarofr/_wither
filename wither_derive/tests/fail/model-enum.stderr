error: only structs can be used as wither models
 --> $DIR/model-enum.rs:5:1
  |
5 | enum BadModel {}
  | ^^^^^^^^^^^^^^^^
